content/8.x/admin/Compatibility Matrix.adoc

```diff
@@ -0,0 +1,49 @@
+---
+Title: Compatibility Matrix
+weight: 60
+---
+[NOTE]
```

Comment:
For the notes this format can be used:
```
{{% notice note %}}
...
{{% /notice %}}
```
The same change can be made in the other files in this PR